include/SFML/Window/Event.hpp

```diff
@@ -154,6 +154,15 @@ class SFML_WINDOW_API Event
         Vector2i position; //!< Position of the mouse pointer, relative to the top left of the owner window
     };
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Mouse move raw event
+    ///
```

Comment:
Can we add some details on when/why this should be used compared to the "normal" mouse events?

Also, if this event is never generated on MacOS, it should be clearly documented here.